### PR TITLE
Improve Timeline's layout

### PR DESCRIPTION
**Pull request by karupayun**
This PR is improving the layout of the Timeline following Carsten
proposition.

Basically:
- We are changing colors.
- We are making the timeline bigger.
- We are centering the labels in the timeline.
- We are fixing an inconsistency between when printing the next label
  and the related tick. Now both are drawn or neither of them.

http://screen/884Hy5d4oonANZ2

Nit: This could have been two different PRs.

http://b/217509785

### CODE REVIEW

File: src/OrbitGl/TimelineUi.cpp [hunk 1]
```diff
@@ -59,15 +59,22 @@ void TimelineUi::RenderLabels(Batcher& batcher, TextRenderer& text_renderer,
     }
     float world_x = timeline_info_interface_->GetWorldFromUs(
         tick_ns / static_cast<double>(kNanosecondsPerMicrosecond));
-    if (world_x > previous_label_end_x) {
+    if (world_x > previous_label_end_x + kLabelMarginRight + kPixelMargin) {
       Vec2 pos, size;
-      float label_bottom_y = GetPos()[1] + GetHeight() - kLabelMarginBottom;
+      float label_bottom_y =
```
Comment (reichlfl) [inline src/OrbitGl/TimelineUi.cpp:64]: You can use `TextRenderer::VAlign::Middle` in the following `AddText` call, then you don't need this calculation.

File: src/OrbitGl/TimelineUi.cpp [hunk 2]
```diff
@@ -59,15 +59,22 @@ void TimelineUi::RenderLabels(Batcher& batcher, TextRenderer& text_renderer,
...
+          GetPos()[1] + (GetHeight() + layout_->GetFontSize()) / 2 - kLabelMarginBottom;
       text_renderer.AddText(label.c_str(), world_x + kLabelMarginLeft, label_bottom_y,
                             GlCanvas::kZValueTimeBar,
                             {layout_->GetFontSize(), Color(255, 255, 255, 255), -1.f,
                              TextRenderer::HAlign::Left, TextRenderer::VAlign::Bottom},
                             &pos, &size);
       previous_label_end_x = pos[0] + size[0];
+
+      // Box between labels and ticks, so they don't overlap.
```
Comment (reichlfl) [inline src/OrbitGl/TimelineUi.cpp:73]: Will this work? Text is drawn after boxes, and text and box are on the same layer, so the label should be drawn on top of the box. Similar to what I stated in the other PR, we really should try to not "hide" elements behind the background color because it'll just cause issues later.

File: src/OrbitGl/TimelineUi.cpp (hunk 1 above)
Comment (karupayun) [reply src/OrbitGl/TimelineUi.cpp:64]: Yes, true! :)

File: src/OrbitGl/TimelineUi.cpp (hunk 2 above)
Comment (karupayun) [reply src/OrbitGl/TimelineUi.cpp:73]: This is currently working, I'm only making it a bit bigger. But yes, we are hiding elements behind background color. Again, a quick discussion could be useful.

**Review by beckerhe — APPROVED** [review]
I can't really judge the implications. But it looks good to me code-wise.

File: src/OrbitGl/TimelineUi.cpp (hunk 2 above)
Comment (reichlfl) [reply src/OrbitGl/TimelineUi.cpp:71]: Ah, I misunderstood - this box is drawn as a text background, I misunderstood the comment. Maybe rephrase the comment to say "Box behind the label to hide the ticks behind it" or something? But now I get it, makes sense.

File: src/OrbitGl/TimelineUi.cpp (hunk 1 above)
Comment (reichlfl) [reply src/OrbitGl/TimelineUi.cpp]: Oh, I just realize this doesn't make it a lot easier because you still need to calculate the middle line... I thought "Middle" allowed you to specify a top and height, and it would automatically center the text in the middle. Anyways, this is fine now.

**Review by reichlfl — APPROVED** [review]
